e2e/etcdctlv3_test.go

```diff
@@ -133,6 +184,36 @@ func ctlV3Get(clus *etcdProcessCluster, key, value string, dialTimeout time.Dura
 	return spawnWithExpects(cmdArgs, key, value)
 }
 
+func ctlV3Watch(clus *etcdProcessCluster, key, value string, dialTimeout time.Duration, afterIdx int, interactive bool) error {
+	if !interactive {
+		cmdArgs := append(ctlV3PrefixArgs(clus, dialTimeout), "watch", key)
```

Comment:
set `cmdArgs` and append `--after-index` in the outer scope before the `interactive` check?